ps_facetedsearch.php

```diff
@@ -265,8 +265,6 @@ public function indexAttribute($idProduct = null)
             ' . ($idProduct === null ? '' : 'AND pa.id_product = ' . (int) $idProduct) . '
             GROUP BY a.id_attribute, pa.id_product , product_attribute_shop.`id_shop`'
         );
-
-        return 1;
```

Comment:
Should'nt we check the index operation went well (either the database execution return or even a READ SQL query to check data looks good) ?

Reply:
hehe, I planned to refactor the installation / upgrade part in the future.

Reply:
Easy answer, "Yes I know, I'll do it in another PR" :trollface: